README.md

```diff
@@ -875,6 +876,7 @@ Table of Contents
   * [ddos-guard.net](https://ddos-guard.net/store/web) — Free CDN, DDoS protection and SSL certificate
   * [developers.google.com](https://developers.google.com/speed/libraries/) — The Google Hosted Libraries is a content distribution network for the most popular, Open Source JavaScript libraries
   * [GraphCDN](https://graphcdn.io/) - Scale, inspect & protect your GraphQL API. 5m requests per month for free.
+  * [GitCDN](https://gitcdn.link) -  A Powerful CDN for Github files
```

Comment:
This one was added recently and I rejected it because all I got from it was heroku error pages. So I'd rather take it out, does not seem reliable.